odl/util/normalize.py

```diff
@@ -262,6 +261,76 @@ def normalized_index_expression(indices, shape, int_to_slice=False):
     return tuple(indices)
 
 
+def normalized_nodes_on_bdry(nodes_on_bdry, length):
```

Comment:
It's in `odl.util` anyway, so it shouldn't be considered completely public. From my point of view, it's okay to have it non-underscored.